src/api/spec/controllers/webui/apidocs_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Webui::ApidocsController, type: :controller do
+  describe "GET index" do
+    describe "correct setup" do
```

Comment:
This should be a context instead of a describe, see: http://betterspecs.org/#contexts